etl/helpers.py

```diff
@@ -647,7 +647,7 @@ def list_all_steps_in_dag(dag: Dict[str, Any]) -> List[str]:
     return all_steps
 
 
-def get_direct_dependencies_for_step_in_dag(dag: Dict[str, Any], step: str) -> Set[str]:
+def get_direct_step_dependencies(dag: Dict[str, Any], step: str) -> List[str]:
```

Comment:
In the future we might wrap all this in its own DAG class (inherit from `Dict[str, Any]`) or even use some existing libraries. (Doing it now would be too much work, but we should keep it in mind)